CONTRIBUTING.md

```diff
@@ -72,3 +72,13 @@ OR
 * The module is under the Node.js foundation Github org
 OR
 * The module is identified as an important module by a Node.js Working Group
+
+#### Procedure
+
+After making sure you adhere to the above requirements, do the
+following:
+
+1. Make sure the `citgm <module>` tests pass (`./bin/citgm.js`)
+1. Add the module to
```

Comment:
Ah, so if I run `citgm module-not-in-lookup` and `citgm module-in-lookup` it behaves differently?